src/com/sun/jna/ptr/ByteByReference.java

```diff
@@ -42,4 +44,8 @@ public byte getValue() {
         return getPointer().getByte(0);
     }
 
+    @Override
+    public String toString() {
+        return String.format("byte@0x%x=0x%x (%d)", Pointer.nativeValue(getPointer()), getValue(), getValue());
```

Comment:
`getValue` needs only to be called once (nitpick):
```
return String.format("byte@0x%1$x=0x%2$x (%4$d)", Pointer.nativeValue(getPointer()), getValue());
```

Reply:
Assuming the last was meant to be `%2$d`?  Good idea.